src/HPWH.cc

```diff
@@ -4910,19 +4911,52 @@ void HPWH::initFromFile(string modelName)
 }
 #endif
 
+//-----------------------------------------------------------------------------
+///	@brief	Determine the inletT testing based on the ambientT
+/// @return	inletT (C)
+//-----------------------------------------------------------------------------
+double findInletT_C(double ambientT_C)
+{
+    const double ambientT_C_E50 = 10.;
+    const double ambientT_C_UEF = 19.7; // EERE-2019-BT-TP-0032-0058, p. 40435
+    const double ambientT_C_E95 = 35.;
+
+    const double inletT_C_E50 = F_TO_C(50.0); // table
+    const double inletT_C_UEF = 14.4;         // EERE-2019-BT-TP-0032-0058, p. 40433
+    const double inletT_C_E95 = F_TO_C(67.0); // table
+
+    if (ambientT_C < ambientT_C_E50)
+        return inletT_C_E50;
+    if (ambientT_C < ambientT_C_UEF)
+    { // linear interp
+        double ratio = (ambientT_C - ambientT_C_E50) / (ambientT_C_UEF - ambientT_C_E50);
+        return (1. - ratio) * inletT_C_E50 + ratio * inletT_C_UEF;
+    }
+    if (ambientT_C < ambientT_C_E95)
+    { // linear interp
+        double ratio = (ambientT_C - ambientT_C_UEF) / (ambientT_C_E95 - ambientT_C_UEF);
+        return (1. - ratio) * inletT_C_UEF + ratio * inletT_C_E95;
+    }
+    return inletT_C_E95;
+}
+
```

Comment:
This really isn't necessary. Each test is a discrete combination of ambient temperature and mains temperature. There are no other tests than those listed in the table provided in the DOE document.

Instead, we can just add additional "TestConfigurations" (see my comment below) if we want to run tests at a different combination of ambient and inlet temperatures.